tests/transport/TestMsg/TestMsg.mpc

```diff
@@ -0,0 +1,2 @@
+project(opendds_tests_transport_testmsg): dcps {
```

Comment:
This new project has an export header, but is missing the `dynamicflags +=` line for it.